prowler/providers/aws/services/opensearch/opensearch_service.py

```diff
@@ -91,12 +91,16 @@ def __describe_domain__(self, regional_clients):
                     DomainName=domain.name
                 )
                 domain.arn = describe_domain["DomainStatus"]["ARN"]
-                domain.endpoint_vpc = None
+                domain.vpc_endpoints = None
                 if "Endpoints" in describe_domain["DomainStatus"]:
                     if "vpc" in describe_domain["DomainStatus"]["Endpoints"]:
-                        domain.endpoint_vpc = describe_domain["DomainStatus"][
-                            "Endpoints"
-                        ]["vpc"]
+                        domain.vpc_endpoints = [
+                            vpc
+                            for vpc in describe_domain["DomainStatus"][
+                                "Endpoints"
+                            ].values()
```

Comment:
Have you check if the `Endpoints` value can have endpoints that are not from VPC?

Reply:
No, the VPC endpoints for an Amazon OpenSearch Service domain must be within the same VPC. However, you can have multiple VPC endpoints within the same VPC to allow access from different subnets within that VPC.


Yet as these are not used by any check should I keep them or remove them?